lib/config/presets/internal/monorepo.ts

```diff
@@ -67,6 +67,16 @@ const repoGroups = {
   commitlint: 'https://github.com/conventional-changelog/commitlint',
   'contentful-rich-text': 'https://github.com/contentful/rich-text',
   'datadog-browser-sdk': 'https://github.com/DataDog/browser-sdk',
+  'datatables-net': [
+    'https://github.com/DataTables/Dist-DataTables',
+    'https://github.com/DataTables/Dist-DataTables-Bootstrap',
+    'https://github.com/DataTables/Dist-DataTables-Bootstrap5',
+    'https://github.com/DataTables/Dist-DataTables-Bulma',
+    'https://github.com/DataTables/Dist-DataTables-DataTables',
+    'https://github.com/DataTables/Dist-DataTables-Foundation',
+    'https://github.com/DataTables/Dist-DataTables-SemanticUI',
+    'https://github.com/DataTables/Dist-DataTables-jQueryUI',
```

Comment:
do they all have same version?

Reply:
These all have the same version.  They are the base component of DataTables.  The others are plugins.